test/numerical/TestLoop.cpp

```diff
@@ -118,9 +118,9 @@ bool isOMLoopTheSameAsNaiveImplFor(std::string moduleIR,
   omTensorGetElem<bool>(condTensor.get(), {}) = true;
   inputs.emplace_back(move(condTensor));
 
-  auto *yInitShape = new int64_t[1]{1};
+  int64_t yInitShape = 1;
   auto yInitTensor = OMTensorUniquePtr(
-      omTensorCreateEmpty(&yInitShape[0], 1, OM_DATA_TYPE::ONNX_TYPE_INT64),
+      omTensorCreateEmpty(&yInitShape, 1, OM_DATA_TYPE::ONNX_TYPE_INT64),
```

Comment:
Can avoid changing this line by changing the declaration for `yInitShape` to:
```
int64_t yInitShape[1] = { 1 };
```
Same for `yRefInitShape` below.

Reply:
Difference is the array definition serves as a reminder that shape is typically an array.

Reply:
makes no difference, but changed it to your liking